### ISSUE TITLE
Feature Request: 

### ISSUE THREAD

**Issue by akariwhim** [dr4fters/dr4ft#1638]
With 2x2 recently releasing, this seems like a good time to bring up the possibility of implementing the "pick two cards on the first pick of a pack" rule used in the double masters sets

**Comment by tooomm** [comment]
This was already discussed (and initially requested) in #1169.
The closing "fix" (#1299) did not address the double masters case though.

**Comment by mixmix** [comment]
Yeah having different numbers picked depending on the pack is quite a bit more to figure out both for application logic, but also UI...

I don't see the effort being worth it for a couple sets personally

**Comment by tooomm** [comment]
To clarify, the quantity of cards being picked doesn't change depending on the pack.
It's always the first pick of all packs being two cards instead of only one card.

Normal:
1-1-1-1-1-1-1-1-1-1-1-1-1-1-1
1-1-1-1-1-1-1-1-1-1-1-1-1-1-1
1-1-1-1-1-1-1-1-1-1-1-1-1-1-1

Double Master
2-1-1-1-1-1-1-1-1-1-1-1-1-1-1
2-1-1-1-1-1-1-1-1-1-1-1-1-1-1
2-1-1-1-1-1-1-1-1-1-1-1-1-1-1

But I do also agree, that it's not high priority.
